model-api-gen/src/main/kotlin/org/modelix/metamodel/generator/internal/ConceptObjectGenerator.kt

```diff
@@ -101,7 +102,7 @@ internal class ConceptObjectGenerator(
     }
 
     private fun TypeSpec.Builder.addConceptPropertiesGetter() {
-        if (concept.metaProperties.isEmpty()) return
+        if (conceptPropertiesInterfaceName == null || concept.metaProperties.isEmpty()) return
```

Comment:
> This was a regression introduced in commit https://github.com/modelix/modelix.core/commit/86c141bdb73b5623c7954ead0de03d1156ed4d6c.

 nit: a test would be nice

Reply:
I agree but it isn't really possible with our current testing setup, since it only allows us to test one single configuration of the plugin.
It's something we have to keep in mind for [MODELIX-798](https://issues.modelix.org/issue/MODELIX-798/Create-a-proposal-for-reorganizing-the-existing-tests)

Reply:
Ok, I added it as an issue in the ticket.